NEWS.md

```diff
@@ -1,3 +1,7 @@
+## 10.0.0 2019-10-03
```

Comment:
The [release guidelines|https://dev.folio.org/guidelines/release-procedures/#prepare-the-news-document] state that the news is not updated within each pull request, please remove this change.

Reply:
done